scripts/status_reporter/aws_client.py

```diff
@@ -0,0 +1,30 @@
+import uuid
+
+import boto3
+
+
+class AwsClient:
+    def __init__(self, role_arn="arn:aws:iam::975596993436:role/storage-developer"):
```

Comment:
Really nice.

We use `attrs` in a couple of places for this sort of thing – sort of similar to case classes in Scala, which makes this a bit cleaner, but not strictly necessary.

Reply:
Should we be using `attrs` as standard practise in this situation?